webpack.config.ts

```diff
@@ -1,27 +1,30 @@
-/* @flow */
 const webpack = require('webpack')
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin')
 const path = require('path')
-const { version, name, author, license } = require('./package.json')
+const { version, author, license } = require('./package.json')
+const packageName = '@userdive/agent'
```

Comment:
if use `name` occur ts error, `cannot redeclare block scoped variable (typescript)` ... 😢 
And I have no idea...